packages/fast-usdc-contract/src/exos/status-manager.ts

```diff
@@ -371,11 +430,35 @@ export const prepareStatusManager = (
         publishTxnRecord(txHash, harden({ split, status: TxStatus.Disbursed }));
       },
 
+      /**
+       * Call when a transaction forward is started.
+       */
+      forwarding(txHash: EvmHash): void {
+        if (forwardInProgress.has(txHash)) {
+          throw Fail`Transaction already in progress: ${q(txHash)}`;
+        }
+        forwardInProgress.add(txHash);
```

Comment:
Should we do a `publishTxnRecord` here with `status: 'Forwarding'`?

Reply:
No I don't think that adds diagnostic value

Reply:
No I don't think that adds diagnostic value